### DISCUSSION TITLE
remove button plugin with custom render broken css

### DISCUSSION THREAD

**Discussion by muhammad-sayed-mahdy** [selectize/selectize.js#1867] [Q&A] [answered]
I did:

* [x] Search for if my issue has already been submitted
* [x] Make sure I'm reporting something precise that needs to be fixed
* [x] Give my issue a descriptive and concise title
* [x] Create a *minimal* working example on JsFiddle or Codepen
	(or gave a link to a demo on the Selectize docs)

Steps to reproduce:
[JS fiddle](https://jsfiddle.net/mwyc01rd/3/)

Expected result:
![image](https://user-images.githubusercontent.com/32793798/181222140-de6806f9-9a7b-4e88-9eb1-4b743d9acbd6.png)

Actual result:
![image](https://user-images.githubusercontent.com/32793798/181222207-ad4698c1-4a1d-4d64-b08c-7eb0e6828e5d.png)

because remove_button plugin assumes `item` class on the outer element of the item, but if you provided a custom render option and didn't provide `item` class, you will get broken css

**Comment by fabienwnklr** [comment, marked as answer]
Hi, is not an issue it's normal, please just add the `item` class to the `div` item

Ex : https://jsfiddle.net/9ec0uf8y/2/
